test/unit/public.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
```

Comment:
looks like all lint fixes..

Reply:
😆   I didn't even notice the changes since they were made automatically by my IDE